src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -538,20 +592,4 @@
         <BitChoiceOption Text="Option2" Value="2"></BitChoiceOption>
         <BitChoiceOption Text="Disabled option" Value="3" IsEnabled="false"></BitChoiceOption>
     </BitChoiceGroup>
-</div>
-<hr />
-<div class="drop-container">
-    <h2>BitDropDown</h2>
-    <br />
-    <h3>Single select</h3>
-        <BitDropDown Items="GetDropdownItems()" Placeholder="Select an option">
-        </BitDropDown>
-        <br />
-     <h3>Disabled</h3>
-        <BitDropDown IsEnabled="false" Items="GetDropdownItems()" Placeholder="Select an option">
-        </BitDropDown>
-        <br />
-    <h3>Multi select</h3>
-    <BitDropDown IsMultiSelect="true" Items="GetDropdownItems()" Placeholder="Select options">
-    </BitDropDown>
```

Comment:
why these lines are removed?

Reply:
I think there're removed in some conflict!
I'll fix it.